tests/client/amo/components/TestLanguagePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import { getFakeI18nInst } from 'tests/client/helpers';
+import I18nProvider from 'core/i18n/Provider';
+
+
+describe('LanguagePicker', () => {
+  function renderLanguagePicker({ ...props }) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <I18nProvider i18n={getFakeI18nInst()}>
```

Comment:
You should be able to pass `i18n` as props instead of using the provider if you test the base component instead. Actually maybe even this component (or if it doesn't work we should make it work).